src/imgs/ppu_modes_timing.svg

```diff
@@ -1,152 +1,307 @@
-	<text class="middle-baseline right" x="55" y="272">2</text>
-	<rect class="scanline m2" y="260" x="60" width="80"/>
-	<rect class="scanline m3" y="260" x="145" width="172"/>
-	<rect class="scanline m30" y="260" x="322" width="112"/>
-	<rect class="scanline m0" y="260" x="439" width="87"/>
-	<text class="middle-baseline small" x="535" y="272">...</text>
-	<text class="middle-baseline right" x="55" y="297">3</text>
-	<rect class="scanline m2" y="285" x="60" width="80"/>
-	<rect class="scanline m3" y="285" x="145" width="172"/>
-	<rect class="scanline m30" y="285" x="322" width="112"/>
-	<rect class="scanline m0" y="285" x="439" width="87"/>
-	<text class="middle-baseline small" x="535" y="297">...</text>
-	<text class="middle-baseline right" x="55" y="322">4</text>
-	<rect class="scanline m2" y="310" x="60" width="80"/>
-	<rect class="square m3" y="315" x="150"/>
-	<rect class="square m3" y="315" x="175"/>
-	<rect class="square m3" y="315" x="200"/>
-	<text class="middle-baseline right" x="55" y="347">...</text>
-	<rect class="square m2" y="340" x="70"/>
-	<rect class="square m2" y="340" x="95"/>
-	<rect class="square m2" y="340" x="120"/>
-	<text class="middle-baseline right" x="55" y="372">...</text>
-	<rect class="square m2" y="365" x="70"/>
-	<text class="middle-baseline right" x="55" y="397">...</text>
-	<rect class="square m2" y="390" x="70"/>
-	<text class="middle-baseline right" x="55" y="422">143</text>
-	<rect class="scanline m2" y="410" x="60" width="80"/>
-	<rect class="scanline m3" y="410" x="145" width="172"/>
-	<rect class="scanline m30" y="410" x="322" width="112"/>
-	<rect class="scanline m0" y="410" x="439" width="87"/>
-	<text class="middle-baseline right" x="55" y="447">144</text>
-	<rect class="scanline m1" y="435" x="60" width="466"/>
-	<text class="middle-baseline right" x="55" y="472">145</text>
-	<rect class="scanline m1" y="460" x="60" width="466"/>
-	<rect class="square m1" y="490" x="70"/>
-	<rect class="square m1" y="515" x="70"/>
-	<rect class="square m1" y="540" x="70"/>
-	<text class="middle-baseline right" x="55" y="572">153</text>
-	<rect class="scanline m1" y="560" x="60" width="466"/>
-	<rect class="m1" y="435" x="533" height="145" width="2"/>
-	<text class="middle-baseline small" x="540" y="490">10 "scanlines"</text>
-	<text class="middle-baseline small" x="540" y="510">4560 dots (= 1140 cycles)</text>
-	<!-- Totals -->
-	<text class="centered" y="20" x="613">One frame:</text>
-	<text class="centered" y="45" x="613">70224 dots</text>
-	<text class="centered" y="70" x="613">(= 17556 cycles)</text>
-	<text class="centered" y="95" x="613">@ 59.7 fps</text>
-</svg>
+<?xml version="1.0" encoding="UTF-8" standalone="no" ?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" width="638" height="580" viewBox="0 0 638 580" xml:space="preserve">
```

Comment:
Without changing [the `include` directive](https://github.com/gbdev/pandocs/blob/eb4c695db1e67f8a999b243b26267225cf609256/src/pixel_fifo.md#L10), this will embed the `<!DOCTYPE>` tag in the document, which is at best harmless cruft, at worst will break *something*.

https://github.com/gbdev/pandocs/blob/eb4c695db1e67f8a999b243b26267225cf609256/src/pixel_fifo.md#L12

Please refer to [the SVG integration guide](https://github.com/gbdev/pandocs/wiki/SVG-guide#dynamically-styled-svgs) for more info.